addon/utils/plugins/table/backspace-plugin.ts

```diff
@@ -0,0 +1,43 @@
+import { BackspacePlugin } from '@lblod/ember-rdfa-editor/editor/input-handlers/backspace-handler';
+import {Manipulation, ManipulationGuidance } from '@lblod/ember-rdfa-editor/editor/input-handlers/manipulation';
+import RawEditor from 'dummy/utils/ce/raw-editor';
+
+/**
+ *
+ * @class TableBackspacePlugin
+ * @module plugins/table
+ */
+export default class TableBackspacePlugin implements BackspacePlugin {
+  label = 'backspace plugin for handling table nodes'
+
+  guidanceForManipulation(manipulation: Manipulation, editor: RawEditor) : ManipulationGuidance | null {
+    const voidExecutor = {
+      allow: false,
+      executor: undefined
+    };
+    const selection = editor.selection;
+    if(selection.isInTable) {
+      if(manipulation.type === 'moveCursorBeforeElement') {
+        return voidExecutor;
+      } else if(manipulation.type === 'removeEmptyTextNode') {
+        if(manipulation.node.parentElement?.childElementCount === 0) {
+          return voidExecutor;
+        } else {
+          return null;
+        }
+      } else {
+        return null;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * If the handler has been executed we had done nothing so we should always return true.
+   * @method detectChange
+   */
+  detectChange() : boolean {
+    return true;
```

Comment:
detectChange overrides default backspace behaviour, so by returning true here you're letting the backspace handler know there is a change even if it did not detect a change itself. note that detectChange is called even if we have not provided guidance.

in this case it's better to always return false I think

Reply:
I thought it only got called when we handled the case. If I return false, it will run in an infinite loop (well, not infinite because it stops after 50 iterations I think) there's no better solution than trying defaulting to false?

Reply:
ah I see! Yes in that case you will need to recognize the specific case (e.g. we're at the start of a cell) and return true only in that case.